icinga2/service.go

```diff
@@ -39,9 +45,9 @@ func (s Service) GetCheckCommand() string {
 	return s.CheckCommand
 }
 
-func (s Service) GetVars() Vars {
-	return s.Vars
-}
+//func (s Service) GetVars() Vars {
+//	return s.Vars
+//}
```

Comment:
As noted above, this needs to implement direct lookup of the Service custom variables from the Icinga2 API if we remove the `Vars` field from the Service struct.